content/spin/contributing-docs.md

```diff
@@ -405,6 +405,21 @@ $ bart check content/about.md
 ✅ content/about.md
 ```
 
+The developer documentation [uses shortcodes](/bartholomew/shortcodes). Always pass `--shortcodes ./shortcodes` as shown below:
```

Comment:
Should we amend the example on line 404?  Non-blocking but that's what the eye will land on first, and I'm not sure I'd read any further... _grin_